roles/osp_observability/tasks/configure_prometheus.yml

```diff
@@ -23,3 +23,12 @@
     mode: '0644'
     dest: "{{ prometheus_config_dir }}/prometheus.yml"
     src: prometheus.yml.j2
+
+- name: "Create CA cert file for remote write endpoint"
+  ansible.builtin.copy:
+    dest: "{{ prometheus_config_basedir }}/etc/prometheus/{{ item.key }}_CA.pem"
+    owner: nobody
+    group: nobody
+    mode: '0644'
+    content: "{{ item.value.ca_cert }}"
+  loop: "{{ prometheus_remote_write | dict2items }}"
```

Comment:
Similar to below comment, will this parameter always be defined (set empty in `defaults.yaml`?). If not, you might need a `when: prometheus_remote_write is defined` on this. I haven't had a chance to test this code as a whole though, this is my starting review.

Reply:
Will this parameter always be defined? In this patch, no, but on #19 I set defaults for all required variables: https://github.com/infrawatch/osp-observability-ansible/pull/19/files#diff-7027c90a62527124195b5351e33095a6077349eaac7e013201e80bb567d2dcb8R4